terraform/account/put_secrets.tf

```diff
@@ -1,147 +1,184 @@
 # common
 
+resource "aws_kms_key" "opg_refunds_secrets_key" {
+  enable_key_rotation = true
+}
+
 
 
 # public front secrets
 resource "aws_secretsmanager_secret" "opg_refunds_public_front_session_encryption_key" {
-  name = "${local.account_name}/opg_refunds_public_front_session_encryption_key"
-  tags = merge(local.default_tags, local.public_front_component_tag)
+  name       = "${local.account_name}/opg_refunds_public_front_session_encryption_key"
+  tags       = merge(local.default_tags, local.public_front_component_tag)
+  kms_key_id = aws_kms_key.opg_refunds_secrets_key.arn
```

Comment:
What's the significance of the kms_key_id entries?

Reply:
these are to tighten up the keys, we now use a Customer Managed Key (not the default AWS one) and is one of the recoomendatins for tfsec. this CMK is auto rotated.